controlpanel/api/models.py

```diff
@@ -50,27 +57,54 @@ def aws_delete_role(self):
         services.delete_role(self.iam_role_name)
 
     def helm_create(self):
-        helm.init_user(self.username, self.email, self.get_full_name())
-        helm.config_user(self.username)
+        user_slug = sanitize_dns_label(self.username)
+        values = ",".join(f"{key}={value}" for key, value in {
```

Comment:
we may want to be more explicit here about the fact that these are values for the `init-user` helm chart, maybe by renaming to `init_user_values`. Not a big deal anyway.

Reply:
Actually, I want to refactor this a bit more, because the levels of abstraction are all wrong. I want to make it so that we can replace Helm if we want to (and I do want to).